appinventor/appengine/src/com/google/appinventor/client/explorer/youngandroid/ProjectListItem.java

```diff
@@ -34,18 +35,18 @@ interface ProjectListItemUiBinder extends UiBinder<FlowPanel, ProjectListItem> {
   private static final ProjectListItemUiBinder UI_BINDER =
```

Comment:
If this is overridden by a subclass, you may want to move it within bindUI otherwise both the classic and neo versions will get bundled together.